xml/System.Windows.Forms/SystemInformation.xml

```diff
@@ -1202,9 +1200,7 @@
  The high-contrast mode feature can be enabled or disabled through the Accessibility Options control panel.  
   
 > [!NOTE]
->  This property is supported only on Windows 98, Windows Millennium Edition, Windows 2000, Windows XP, and the Windows Server 2003 family.  
-  
-   
+> This property is supported only on Windows 2000, Windows XP, and the Windows Server 2003 family.
```

Comment:
For this and other properties, I believe the original text was meant to imply "or later", which means that the property is supported on all in-market versions of Windows. There are still references to it in the public win32 documentation (https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-systemparametersinfoa). Somebody from WinForms should double-check me on this.

Reply:
From the Win32 docs it isn't obvious. 

E.g.
  ![image](https://user-images.githubusercontent.com/4403806/112098312-ad8ed700-8bf5-11eb-8ddd-d868f9c76efd.png)

Does it mean:
- it is not supported _only_ on W2000, -or-
- it is not supported W2000 and later (XP, etc), -or-
- it is not supported W2000 and earlier (9x)?

Reply:
Checked WinUsr.h file and it looks like the docs should indeed say "**Windows 2000 _and earlier_**:  This parameter is not supported."